fetch-config.sh

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env bash
+
+aws s3 cp s3://membership-private/DEV/support-frontend.private.conf \
```

Comment:
Need to ensure you have permission to write to `/etc/gu`.

Reply:
Done.